core-bundle/src/Resources/contao/dca/tl_member.php

```diff
@@ -111,7 +111,7 @@
 	// Subpalettes
 	'subpalettes' => array
 	(
-		'login'                       => 'username,password',
+		'login'                       => 'useTwoFactor,username,password',
```

Comment:
Have not checked the back end yet, but should the 2FA checkbox not be after username/password?

Reply:
Probably yes. Will check this during merge.

Reply:
Fixed in 6c91c392a.